wire-ios-data-model/Source/Model/Conversation/ZMConversation+SecurityLevel.swift

```diff
@@ -435,9 +435,9 @@ extension ZMConversation {
 
     /// Accepts the privacy changes (legal hold and/or degradation) and resend the pending messages.
     @objc(acknowledgePrivacyWarningWithResendIntent:) public func acknowledgePrivacyWarning(withResendIntent shouldResendMessages: Bool) {
-        acknowledgePrivacyChanges()
 
         if shouldResendMessages {
+            acknowledgePrivacyChanges()
```

Comment:
why do we acknowledge only when we resend? I guess then we should change the name of the method or of its parameter

Reply:
Should we even call this method the user taps cancel? I thought the user did not accept the degrade if they press cancel.